.gitignore

```diff
@@ -1 +1,4 @@
 vendor
+composer.lock
```

Comment:
it is autogenerated from composer.json when you install dependencies, so i dont see why it should be in the repository. but if you prefer it that way i can re-add it